docs/accounts/add_another.md

```diff
@@ -0,0 +1,13 @@
+# Adding another account
+
+Once you have a first account set up, you can add another using the settings icon at the bottom
+of the screen.
+
+![settings icon](../img/screenshots/settings_select.png)
+
+Select add account:
```

Comment:
I think it would be helpful to mark the part of the sentence that refers to the text on the screen. In other parts of the documentation we use double quotes and retain the capitalization from the UI.
```
Select "Add account":
```

I don't mind changing the style, e.g. italicize such strings. But it should be used consistently.

Reply:
fixed